skyvern/experimentation/wait_config.py

```
@@ -0,0 +1,201 @@
+    if payload:
+        try:
+            config_dict = json.loads(payload) if isinstance(payload, str) else payload
+            wait_config = WaitConfig(config_dict)
+            LOG.info(
+                "Wait time optimization experiment enabled",
+                distinct_id=distinct_id,
+                organization_id=organization_id,
+                variant=wait_optimization_experiment,
+                config=wait_config.to_dict(),
+            )
+            return wait_config
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)